astro/src/content/docs/apis/message-templates.mdx

```diff
@@ -138,7 +138,18 @@ This API does not return a JSON response body.
 
 This API is used to preview a Message Template. You pass all of the information for the Message Template in the request and a rendered version of the Message is sent back to you in the response.
 
-FusionAuth provides sample values for the `${code}`, `${verificationId}`, and `${verificationOneTimeCode}` parameters when previewing. The values for these parameters will always be `123456`, `987654`, and `abcdef` respectively. The `${code}` parameter is used to deliver a two factor code to the user while `${verificationId}` and `${verificationOneTimeCode}` are used for phone number verification messages.
+FusionAuth provides sample values for the following parameters when previewing:
+
```

Comment:
What do you think about moving this to the customization section and referencing it from here? Reason being that it's useful for customizing message templates, regardless of the method (API or UI), and this area is specific to the API.

Reply:
That probably would make more sense.

Reply:
Pushed as 3381ad439271f31034016e9ab6079cde8e616a99

Reply:
Had to move around again because issue 1 doc branch changed all of this during this PR - see  merge commit 97b7051240e179f489a90c2be7d09d15b220c27c and adaptation in 0bbc53c0b97fd0fddfe43d2977543cb5147d790f